src/brogue/Recordings.c

```diff
@@ -1114,9 +1123,10 @@ void switchToPlaying() {
     rogue.playbackOmniscience   = false;
     locationInRecordingBuffer   = 0;
     copyFile(currentFilePath, lastGamePath, recordingLocation);
-
-#ifdef DELETE_SAVE_FILE_AFTER_LOADING
-    remove(currentFilePath);
+#ifndef ENABLE_PLAYBACK_SWITCH
+    if(DELETE_SAVE_FILE_AFTER_LOADING) {
+      remove(currentFilePath);
+    }
```

Comment:
Is there are particular reason this was changed to a conditional from an `#ifdef`?

If this is to remain an `if`, the body is only indented 2 spaces instead of 4. Also `if(` -> `if (`

```suggestion
    if (DELETE_SAVE_FILE_AFTER_LOADING) {
        remove(currentFilePath);
    }
```

Reply:
DELETE_SAVE_FILE_AFTER_LOADING is defined to either true or false in Rogue.h, so the ifdef would have always executed.